scripts/v.import/v.import.py

```diff
@@ -208,8 +252,12 @@ def main():
     # create temp location from input without import
     grass.verbose(_("Creating temporary location for <%s>...") % OGRdatasource)
     try:
+        if checkGDALversion():
+            if os.path.basename(OGRdatasource).split('.')[-1].lower() == "gml":
```

Comment:
`os.path.splitext()` is splitting the extension so shorter and easier to read, but you can consider `.lower().endswith("gml")` which might be even more readable and thus better Python.